security/crowdsec/files/crowdsec.in

```diff
@@ -35,14 +35,32 @@ configtest_cmd="${name}_configtest"
 extra_commands="configtest reload"
 
 crowdsec_precmd() {
-    if [ ! -d %%PREFIX%%/etc/crowdsec/hub ]; then
-        %%PREFIX%%/bin/crowdsec-cli hub update || :
+    HUB_DIR=`%%PREFIX%%/bin/cscli config show | grep 'Hub Folder' | head -1 | awk -F: '{gsub(/: /,":"); printf "%s", $2}'`
```

Comment:
Can you just limit number of pipes here using just `awk` or `sed`?  or maybe you can rely on the output `-o raw` to ease the parsing of the config value.

Reply:
I know, I couldn't do it simpler and the configuration option could appear twice in the output. Upon my request, the next version of CrowdSec has a "config show <specific_option>" to display only what I want, so all of those pipes will go away.